fence-agents.spec.in

```diff
@@ -55,6 +55,7 @@ fence-agents-docker \\
 fence-agents-drac \\
 fence-agents-drac5 \\
 fence-agents-eaton-snmp \\
+fence-agents-eaton-ssh \\
```

Comment:
You also need to add an eaton-ssh section. Just copy the eaton-snmp section and update the description and requirements.

Reply:
Thanks for the hint. That was an oversight of me. Should be fixed.